python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -963,3 +963,26 @@ def _assert_in_rnn_block_(self, method):
         if self.status != DynamicRNN.IN_RNN:
             raise ValueError("{0} can only be invoked inside rnn block.".format(
                 method))
+
+
+def reorder_lod_tensor_by_rank(x, rank_table):
+    """
```

Comment:
Please refine the doc.

Reply:
I think we could use python to generate the docstring.

Reply:
Done